bubuku/features/rebalance/change.py

```diff
@@ -134,11 +135,11 @@ def run(self, current_actions) -> bool:
             return True
         if self.zk.is_rebalancing():
             return True
-        new_broker_ids = sorted(int(id_) for id_ in self.zk.get_broker_ids())
-        if new_broker_ids != self.broker_ids:
-            _LOG.warning("Rebalance stopped because of broker list change from {} to {}".format(
-                self.broker_ids, new_broker_ids))
-            return False
+        # new_broker_ids = sorted(int(id_) for id_ in self.zk.get_broker_ids())
+        # if new_broker_ids != self.broker_ids:
+        #     _LOG.warning("Rebalance stopped because of broker list change from {} to {}".format(
+        #         self.broker_ids, new_broker_ids))
+        #     return False
```

Comment:
If I understand correctly, it's ok to keep it even when the process is manually started?